lib/Backends.hs

```diff
@@ -60,5 +81,6 @@ backendPackedCodec = Toml.Codec input output
 
 supportedBackends
   :: Text -> Either Toml.TomlBiMapError (Toml.TomlEnv SomeBackend)
-supportedBackends "vault-kv" = Right $ fmap SomeBackend . Toml.codecRead (_codec @VaultKvBackend)
-supportedBackends _ = Left (Toml.ArbitraryError "Unknown backend type")
+supportedBackends backend = case HS.lookup (T.unpack backend) supportedBackendsMap of
+  Just x -> Right (bToml x)
+  Nothing ->  Left (Toml.ArbitraryError "Unknown backend type")
```

Comment:
I think at this point it makes sense to inline `supportedBackends` into `backendPackedCodec`.

(And, while we're here, rename `backendPackedCodec` to `someBackendCodec`, seems like a more appropriate name)

Reply:
done
I haven't get rid of supportedBackends completely 
it was the best way to save initial structure